salt/metalk8s/orchestrate/solutions/files/operator/deployment.yaml

```diff
@@ -3,15 +3,29 @@ kind: Deployment
 metadata:
   name: {{ solution }}-operator
   namespace: {{ namespace }}
+  labels:
+    app: {{ solution }}-operator
+    app.kubernetes.io/name: {{ solution }}-operator
+    app.kubernetes.io/instance: {{ solution }}-operator
+    app.kubernetes.io/version: {{ version }}
+    app.kubernetes.io/component: operator
+    app.kubernetes.io/part-of: {{ solution }}
+    app.kubernetes.io/managed-by: salt
 spec:
   replicas: 1
   selector:
     matchLabels:
-      name: {{ solution }}-operator
+      app: {{ solution }}-operator
```

Comment:
I think this change here will break the Solutions test.
It is related to this line and must be fixed
`        Then we have 1 running pod labeled 'name=example-solution-operator' in namespace 'example-environment'
`